kclvm/config/src/modfile.rs

```diff
@@ -107,9 +107,9 @@ pub fn get_pkg_root_from_paths(file_paths: &[String]) -> Result<String, String>
     }
     if m.len() == 1 {
         return Ok(last_root);
+    } else {
+        return Ok(work_dir);
```

Comment:
It is necessary to determine whether workdir is empty here, as workdir is inherently optional and will not be inputted at the API function level.